### ISSUE TITLE
IdentifierMapper crashes on Ctrl+C

### ISSUE THREAD

**Issue by rkm** [SMI/SmiServices#131]
Found with version `1.4.4` on the live system:

```bash
$ dotnet ../SmiServices/IdentifierMapper.dll -y ../configs/smi_dataLoad_mysql.yaml

...
^C
Unhandled Exception: RabbitMQ.Client.Exceptions.AlreadyClosedException: Already closed: The AMQP operation was interrupted: AMQP close-reason, initiated by Application, code=200, text='Dispo
sed', classId=0, methodId=0
   at RabbitMQ.Client.Impl.SessionBase.Transmit(Command cmd)
   at Smi.Common.Messaging.Consumer.Ack(IMessageHeader header, BasicDeliverEventArgs deliverEventArgs) in /home/travis/build/SMI/SmiServices/src/common/Smi.Common/Messaging/Consumer.cs:line 
168
   at Microservices.IdentifierMapper.Messaging.IdentifierMapperQueueConsumer.<.ctor>b__9_0() in /home/travis/build/SMI/SmiServices/src/microservices/Microservices.IdentifierMapper/Messaging/
IdentifierMapperQueueConsumer.cs:line 49
   at System.Threading.Thread.ThreadMain_ThreadStart()
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
Aborted (core dumped)
```

**Comment by tznind** [comment]
Almost certainly the microservice trying to ack after shutdown given the multi threading going on in that microservice it's not too surprising.  Probably need a cancellation token to be injected from further up @jas88 .

**Comment by jas88** [comment]
Fixed on feature branch, just waiting for Travis run to check it cleans up fully now. (Probably time I asked Emily about budget for getting a PC I can run/debug code on instead of relying on Travis runs...)

**Comment by jas88** [comment]
Resolved by #132
